.github/actions/l10n-update-pot/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Update *.pot files
+
+# This action expects to have these repositories checked out:
+#   in directory `l10n` is the repository with weblate translations (e.g. dnf-l10n)
+#   in directory `source` is repository with sources (e.g. dnf)
+
+inputs:
+  branch:
+    required: true
+    description: Subdirectory where to store translations
```

Comment:
The argument is called `brach`, but the description talks about a subdirectory, so this might need a bit more explaining in the description? e.g.
> The branch from source repository, which is a directory in the l10n repository in which translations for the branch are stored